### ISSUE TITLE
Windows installation failes complaining about GTK3 and TkInter

### ISSUE THREAD

**Issue by ralemons** [textext/textext#262]
**Description of the problem/ steps to reproduce**
A clear and concise description of what the bug is and what you expected to happen.

When I double click the setup_win.bat it runs and finds inkscape and latex but fails at finding the GUI library OR GTK3 OR TkInter.

**Steps to reproduce the behavior (if not obvious from description)**:
1. ....
2. ....
3. ....

**Message(s) in error dialog (if this comes up)**
Please paste the error message(s) at the marked positions:

<details>
<summary>dialog output (if available):</summary>

```[TexText][CRITICAL]: + [Fail] TexText requirements
[TexText][SUCCESS ]: /-and-* [Succ] Detect inkscape>=1.0
[TexText][SUCCESS ]: |      inkscape=1.0 is found
[TexText][SUCCESS ]: /-and-+ [Succ] Detect *latex
[TexText][SUCCESS ]: |     /--or-* [Succ] `pdflatex.exe` is found at `C:\Users\USERNAME\AppData\Local\Programs\MiKTeX\miktex\bin\x64\`
[TexText][SUCCESS ]: |     /--or-* [Succ] `lualatex.exe` is found at `C:\Users\USERNAME\AppData\Local\Programs\MiKTeX\miktex\bin\x64\`
[TexText][SUCCESS ]: |     /--or-* [Succ] `xelatex.exe` is found at `C:\Users\USERNAME\AppData\Local\Programs\MiKTeX\miktex\bin\x64\`
[TexText][CRITICAL]: /-and-+ [Fail] Detect GUI library
[TexText][CRITICAL]:        Please follow installation instructions at
[TexText][CRITICAL]:               https://textext.github.io/textext/install/windows.html#windows-install-gui-library
[TexText][CRITICAL]:       /--or-* [Fail] GTK3 is not found
[TexText][CRITICAL]:       |      Please follow installation instructions at
[TexText][CRITICAL]:       |             https://textext.github.io/textext/install/windows.html#windows-install-gtk3
[TexText][CRITICAL]:       /--or-* [Fail] TkInter is not found
[TexText][CRITICAL]:              Please follow installation instructions at
[TexText][CRITICAL]:                     https://textext.github.io/textext/install/windows.html#windows-install-tkinter
[TexText][ERROR   ]: Automatic requirements check found issue
[TexText][ERROR   ]: Follow instruction above and run install script again
[TexText][ERROR   ]: To bypass requirement check pass `--skip-requirements-check` to setup.py
```
</details>

<details>
<summary>output of stdout (if available):</summary>

```
<<< REPLACE THIS LINE WITH ACTUAL OUTPUT >>> 
```
</details>

<details>
<summary>output of stderr (if available):</summary>

```
<<< REPLACE THIS LINE WITH ACTUAL OUTPUT >>> 
```
</details>

**Logfile**
Please also *attach* the TexText log file (since TexText 0.9.x). You find it in `~/.config/inkscape/extensions/textext` on Linux/ MacOS or in `C:\Users\[Your user name]\AppData\Roaming\inkscape\extensions\textext` on Windows

There is no log file as it hasn't installed yet

**Screenshots**
If applicable and helpful, add screenshots to help explain your problem.

**System information**

- TexText version: 1.1.0
- Inkscape version: 1.0.1
- Operating system: Windows 10 64 bit
- Windows only: LaTeX-distribution (MiKTeX, TeX-Live): MikTeX
- Windows only: TexText installed via batch script or installer file: batch script

**Additional information if you still use Inkscape 0.92.x / TexText 0.11**

- SVG-converter installed (pstoedit+ghostscript or pdf2svg):
- GUI framework installed (PyGTK, PyGTK+PyGTK-Sourceview, TkInter):

**Comment by jcwinkler** [comment]
@ralemons When you invoke the batch script at the very beginning some lines like this should be printed:

![grafik](https://user-images.githubusercontent.com/11866252/96864049-1c5ddd80-1468-11eb-9eb9-97ffea2a1e4a.png)

What is shown on your system?

**Comment by ralemons** [comment]
![bat_ouput](https://user-images.githubusercontent.com/46497943/97033309-00b21000-1518-11eb-9682-06047131468e.PNG)

This is the full output of the bat script and MikTeX is up to date

**Comment by jcwinkler** [comment]
I am quite surprised by this error because Inkscape and the Python subsystem are correctly detected but not the gui bindings.

Maybe you could do some manual testing, if possible. 

1. Open a command prompt in `c:\Program Files\Inkscape\bin\`

2. In that directory call python from this command prompt

3. Enter the following commands in the Python interpreter:
   ```python
   import gi
   gi.require_version("Gtk", "3.0")
   from gi.repository import Gtk
   from gi.repository import Gdk
   from gi.repository import GdkPixbuf
   ```
Which one of these fails?

Also a complete re-install (with double cheking that `C:\Program Files\Inkscpae` is completey removed in between) might by worth trying.

**Comment by ralemons** [comment]
TL:DR I think this is an issue with the 1.0.1 version of Inkscape and the quick fix is installing Inkscape 1.0 instead of the latest release.

We are in luck, sort of, I did two wrongs and they made a right. I was in a rush this morning and forgot to collect the output of the cmd commands which failed on the "Gtk" and "Gdk" imports but not "GdkPixbuf". I then uninstalled Inkscape 1.0.1 and reinstalled Inkscape _1.0_ accidentally instead of 1.0.1. When I then ran through the textext installation steps, it all worked just fine. This surprised me and that's when I noticed the version change. So then I uninstalled 1.0, re-installed 1.0.1, and went through the installation steps for textext with the same failures as before and remembered to collect the cmd outputs (which were the same as the first time) this time around.

![cmd_output](https://user-images.githubusercontent.com/46497943/97912328-85afdd00-1d01-11eb-97d1-a5087a2eaaee.PNG)

It seems like the .`..\gi\module.py` file is trying to find a file that can't be found. I am not sure if the fix here lies with Inkscape or you however.

**Comment by jcwinkler** [comment]
The message about missing gtk3 bindings is not true. Is is an error by the installer. It is related to #255. 

**Short:** Install Inkscape 1.0.1, install TexText 1.2, ignore warning message or call with `setup_win --skip-requirements-check`. Launch Inkscape and enjoy.

**Long**: TexText 1.2 is shipped together with the harfbuzz lib missing in the Inkscape 1.0.1 package. At the moment of installation this lib is not known to the environment. That's why setup complains. However, when TexText 1.2 is installed in Inkscape 1.0.1 the missing lib available.

I will provide a fix for this misleading behaviour of the setup.

**Comment by jcwinkler** [comment]
@ralemons The issue should be fixed now in the most recent [development release](https://github.com/textext/textext/releases).

Any feedback is highly appreciated.
